src/exceptions.ts

```diff
@@ -390,7 +390,7 @@ export class AppException<ErrorCode extends ERROR_CODE> extends HttpException {
       {
         errorCode: code,
         error: (extraMessages as string[]).reduce(
-          (message, extra) => message.replaceAll('%', extra),
+          (message, extra) => message.replace('%', extra),
```

Comment:
ça gêne de mettre replaceAll ? (qui est là au cas où l'implémentation nodejs changerait pour mieux implémenter replace) - same pour test/declarations.ts

Reply:
si extraMessages = ["a", "b"] et message = "ici a : %, ici b : %"
pour extra = "a" : message = "ici a : a, ici b : a"
pour extra = "b" : message = "ici a : a, ici b : a"

==> on voulait "ici a : a, ici b : b"

Reply:
nope, ya un reduce avant, on itère sur tous les extraMessages